cypress/integration/visual-test/toggle-test.spec.js

```diff
@@ -0,0 +1,21 @@
+import { baseURL, checkEyes } from '../../util';
+describe('Visual Regression Testing', () => {
+  before(() => {
+    cy.eyesOpen({
+      appName: 'Clarity',
+      testName: 'toggles',
+      browser: { width: 1024, height: 768, name: 'chrome' },
+    });
+  });
+  after(() => {
+    cy.eyesClose();
+  });
+  it('check toggle-button', () => {
+    cy.visit(baseURL + 'toggles');
+    checkEyes('toggle off');
+    cy
+      .get('[clrlayout=vertical] .clr-form-control:first-child .clr-toggle-wrapper:first-child .clr-control-label')
```

Comment:
What would be _really awesome_ here is to click on all of the inputs to toggle them on and then snapshot.

Reply:
Done
![toggle](https://user-images.githubusercontent.com/22325351/54480972-1dd71680-47ec-11e9-8643-a066867f2740.png)